debian/kafka/Dockerfile

```diff
@@ -29,12 +29,15 @@ ENV KAFKA_ZOOKEEPER_CONNECT=${KAFKA_ZOOKEEPER_CONNECT}
 ARG KAFKA_ADVERTISED_LISTENERS
 ENV KAFKA_ADVERTISED_LISTENERS=${KAFKA_ADVERTISED_LISTENERS}
 
+ARG RELEASE_QUALITY=${RELEASE_QUALITY}
+
 ENV COMPONENT=kafka
 
 # primary
 EXPOSE 9092
 
 RUN echo "===> installing ${COMPONENT}..." \
+    && if [ "${RELEASE_QUALITY}" = "beta" ] || [ "${RELEASE_QUALITY}" = "rc" ]; then CONFLUENT_PLATFORM_LABEL=""; fi \
```

Comment:
CONFLUENT_PLATFORM_LABEL isn't set for RC in cp-docker-images-overlay, so can u explain why u need to set the value to empty for that case?

As for beta, that should be using the -betaNNN label so it shouldn't be empty as we do build Kafka?

Reply:
For BETA, -betaNNN fails for kafka and zookeeper as its unable to find the label during  apt-get install -y confluent-kafka-${SCALA_VERSION}=${KAFKA_VERSION}${CONFLUENT_PLATFORM_LABEL}-${CONFLUENT_DEB_VERSION}.

For RC, I think i will remove it for now & will re-visit this again during 4.1.2 rc. Looks like RC would also be needing this. I had to do a bunch of similar changes during last rc.